test/harry/main/org/apache/cassandra/harry/gen/SchemaGenerators.java

```diff
@@ -94,4 +96,16 @@ public ColumnSpec<?> generate(EntropySource rng)
             }
         };
     }
+
+    public static LongFunction<SchemaSpec> trivialSchema(String ks, String table, int population)
```

Comment:
can we use `Generator` instead?  the class name is `Generator`, all other methods use `Generator`, so this becomes an outlier.  Even the caller has to do `rng.next()` rather than `rng` like the other code paths